reference/pipeline/stages/index.md

```diff
@@ -13,10 +13,24 @@ sidebar:
 This article describes the currently-supported stages that you can add to your
 Spinnaker pipelines.
 
+Note that when you're creating a pipeline, you probably won't see every stage
+that's listed here: you'll only see the stages that Spinnaker supports on your
+provider.
+
 ## General
 
 ### Bake
-Bake an image in the specified region.
+Bake an image from the specified packages. _Baking_ is the process of
+creating a machine image. Spinnaker provides default
```

Comment:
I might call out that Spinnaker happens to use Packer to bake images, rather than assume that it's known why default Packer templates are provided.